### ISSUE TITLE
DrawableD2D error

### ISSUE THREAD

**Issue by m4tt86** [lbmonsalve/dbreport-website#1]
On preview page I got this error:

Common\Win32\DrawableD2D.cpp:441
Failure Condition:
SUCCEDED(render Target->CreateBitmapFromWicBitmap(bitmapWith Mask, nullptr, &bitmap))

**Comment by lbmonsalve** [comment]
what version of xojo, windows10 and DBReport.Version use? 

have issues on xojo2019x / win10, the users have to downgrade xojo to 2018/2017

**Comment by m4tt86** [comment]
Hello,

Windows 7 (with service pack 1)
Xojo 2018 Release 2




Daniele Murabito

-------------------------------------------
Worksdem
love for the future
www.worksdem.com <http://www.worksdem.com/>

Via Giovanni Verga, 39
95016 - Mascali (CT)
Tel: +39 095 7791140
E-mail: info@worksdem.com <mailto:info@worksdem.com>
Skype: worksdem

**Comment by m4tt86** [comment]
> what version of xojo, windows10 and DBReport.Version use?
> 
> have issues on xojo2019x / win10, the users have to downgrade xojo to 2018/2017

Windows 7 (with service pack 1) Xojo 2018 Release 2

**Comment by lbmonsalve** [comment]
ok, I have a machine with 2018r4, I'll download r2 and test it, please give some time to test. the r4 works fine in debug, you can test the "DBReport example" proyect? just for check it.

**Comment by m4tt86** [comment]
Hello, any news about my problem? I tried DBReport example with same result.
I also installed xojo 2018r3 same error.

**Comment by lbmonsalve** [comment]
Well, I cant reproduce the error, I have a Win7 sp1 machine with xojo 2014r2.1, 2017, 2018r4, 2019r1.1 with no error at all, others users report [same error](https://forum.xojo.com/51504-runtime-error-drawabled2d/0#p417324) to xojo (not my code), is full screen in the preview? what screen resolution do you have?

**Comment by m4tt86** [comment]
Hello, I think that the error is related to your plugin because if I use DBReport 2015 it works very well if I use DBReport 2017 or 2018 I get the error. 
I tried your "DBReport example" with xojo 2018r3 and DBReport 2018 and I get the same error.

**Comment by lbmonsalve** [comment]
Yes, could be. You can test with 2018r4? I have VM with 2018r2 with no issues, I can't reproduce the error, I know the issue is real, but I don't know what line causes the error, that is my problem and I said before, other users report same error with other but DBReport code, again, is full screen in the preview? what screen resolution do you have?
